processor/aggregationprocessor/README.md

```diff
@@ -0,0 +1,141 @@
+# Aggregation Processor
+This processor aggregates metrics over a configurable interval, allowing for metrics to be sampled at a higher rate, or to reduce the volume of metric data from push-based sources.
+
+## Minimum collector versions
+- Introduced: [v1.18.0](https://github.com/observIQ/observiq-otel-collector/releases/tag/v1.18.0)
+
+## Supported pipelines
+- Metrics
+
+## How it works
+1. The user configures the resource attribute transposer processor in the desired metrics pipeline.
+2. Every metric that flows through the pipeline is matched against the provided `include` regex.
+3. If the metric name does not match, the metric passes through the processor.
+4. If the metric is not a gauge or cumulative sum, the metric passes through the processor.
```

Comment:
Does this mean if it matches but it's not a gauge?

Reply:
Updated the wording here to be more clear